histomicsui/web_client/views/body/ImageView.js

```diff
@@ -1172,6 +1172,24 @@ var ImageView = View.extend({
                     this.drawWidget.setToNextStyleGroup();
                 }
                 break;
+            case 'Enter':
+                if (this.annotationSelector._activeAnnotation && (
+                    this.drawWidget._drawingType === 'polygon' || this.drawWidget._drawingType === 'line')
+                ) {
+                    const annotation = this.viewerWidget.annotationLayer.annotations()[0];
```

Comment:
@manthey I made the assumption here that the truthiness of `this.annotationSelector._activeAnnotation` implies `this.viewerWidget.annotationLayer.annotations()` has at least one element - is that correct, or should I add a guard here to ensure `this.viewerWidget.annotationLayer.annotations()` has at least one element?